src/content/docs/fr/guides/styling.mdx

```diff
@@ -484,6 +484,27 @@ npm install less
 
 Utilisez `<style lang="less">` dans les fichiers `.astro`.
 
+### LightningCSS
+
+```shell
+npm install lightningcss
+```
+
+Mettez à jour votre configuration `vite` dans `astro.config.mjs` :
+
+```js title="astro.config.mjs" ins={4-8}
+import { defineConfig } from 'astro/config'
+
+export default defineConfig({
+  vite: {
+    css: {
+      transformer: "lightningcss",
+    },
+  },
+})
+
+```
+
```

Comment:
Can you add these changes recently [added](https://github.com/withastro/docs/commit/752278ad39b198fa40490d4c42acaf8d7d131976)

Reply:
Hey @dreyfus92, thanks for your notification about this recent edition. I commit that in https://github.com/withastro/docs/pull/7109/commits/c372ed104d6e636da940f6f75e8199692583efab